manim/mobject/types/vectorized_mobject.py

```diff
@@ -922,18 +953,38 @@ def __init__(self, *pairs, show_keys=False, **kwargs):
         
 
     def add(self, *pairs):
-        '''Adds the key-value pairs to self.submob_dict
-            and the value to the 'submobjects' list of Mobject,
-            (which is responsible for actual on-screen display)
+        '''Adds the key-value pairs to the :class:`VDict` object.
+
+            Also, it internally adds the value to the `submobjects` :class:`list`
+            of :class:`~.Mobject`, which is responsible for actual on-screen display
+            
+            Paramters
```

Comment:
```suggestion
            Parameters
```
Just a tiny typo.

Everything else LGTM, but @PgBiel might have something to say about the docs.

Reply:
Okay great. I'll wait for @PgBiel comments on docs.